src/Tables/class_based_pokeball_table.c

```diff
@@ -69,7 +69,7 @@ const u8 gClassPokeBalls[NUM_TRAINER_CLASSES] =
 	[CLASS_YOUNG_COUPLE_RS] = BALL_TYPE_POKE_BALL,
 	[CLASS_OLD_COUPLE] =      BALL_TYPE_POKE_BALL,
 	[CLASS_SIS_AND_BRO_RS] =  BALL_TYPE_POKE_BALL,
-	[CLASS_AQUA_ADMIN] =      BALL_TYPE_DIVE_BALL,
+	[CLASS_PLUTO_ADMIN] =      BALL_TYPE_MOON_BALL,
```

Comment:
is there an extra space here?

Reply:
Yes, good catch!  I pushed up a formatting fix